pkg/resources/resources.go

```diff
@@ -746,6 +746,8 @@ const (
 	ServiceAccountTokenAnnotation = "kubernetes.io/service-account.name"
 
 	UserSSHKeys = "usersshkeys"
+
+	TinkerbellKubeconfig = "kubeConfig"
```

Comment:
Please document this constant. What is it used for?

Reply:
Go comments should be of the form "<Name> ....", like "// TinkerbellKubeconfig is used...".

Can we change the constant value to be lowercase or is that set in stone already?